cmd/registry.go

```diff
@@ -72,16 +105,11 @@ var registryListCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(registryCmd)
 	// Registry Add Flags
-	registryAddCmd.Flags().StringVar(&registryConfig.Config.Type, "type", "dockerhub", "Type of registry adapter to add")
-	registryAddCmd.Flags().StringVar(&registryConfig.Config.Org, "org", "ansibleplaybookbundle", "Type of registry adapter to add")
-	registryAddCmd.Flags().StringVar(&registryConfig.Config.URL, "url", "docker.io", "URL of registry adapter to add")
-	registryAddCmd.Flags().StringVar(&registryConfig.Config.Name, "name", "docker", "Name of registry adapter to add")
-	registryAddCmd.Flags().StringVar(&whitelist, "whitelist", ".*-apb$", "Whitelist for configuration of registry adapter")
```

Comment:
Ah we used to create a registry config before, why don't we store the overrides in a registryConfig, then use that to apply it to the REAL config before saving?

Reply:
Done :)